messaging/integration_test/src/integration_test.cc

```diff
@@ -361,6 +361,29 @@ TEST_F(FirebaseMessagingTest, TestReceiveToken) {
   EXPECT_TRUE(WaitForToken());
   EXPECT_NE(*shared_token_, "");
 
+  if (*shared_token_ == "") {
+    ::firebase::messaging::Terminate();
+    ::firebase::messaging::Initialize(*shared_app_, shared_listener_,
+                                      firebase::messaging::MessagingOptions());
+  }
+
+  FLAKY_TEST_SECTION_END();
+}
+
+TEST_F(FirebaseMessagingTest, TestRequestToken) {
```

Comment:
should this be `TestReceiveToken`?

Reply:
I'd rather call this request, since it's started the call to receive token.